xbmc/platform/linux/WebOSTVPlatformConfig.h

```diff
@@ -8,13 +8,33 @@
 
 #pragma once
 
+#include <atomic>
+#include <memory>
+
+#include <webos-helpers/libhelpers.h>
+
 class CVariant;
 
+struct HContextDeleter
+{
+  void operator()(HContext* ctx) const noexcept
+  {
+    if (ctx)
```

Comment:
This is not necessary, as the destructor is only called if the unique_ptr is not null